File: quiz/src/main/java/com/example/quiz/service/question_group/QuestionGroupService.java
package com.example.quiz.service.question_group;

import com.example.quiz.base.baseInterface.BaseService;
import com.example.quiz.model.entity.question_group.QuestionGroup;
import com.example.quiz.model.entity.question_group.QuestionGroupRequest;
import com.example.quiz.model.entity.question_group.QuestionGroupResponse;
import com.example.quiz.model.entity.question_group.QuestionGroupView;

public interface QuestionGroupService extends BaseService<QuestionGroup, Long, QuestionGroupRequest, QuestionGroupResponse, QuestionGroupView> {
}

File: quiz-fe/src/services/question_group/question-group.service.ts
import { ENV } from "@/config/env";
import { BaseService } from "../BaseService";
import { QuestionGroup, QuestionGroupRequest, QuestionGroupResponse, QuestionGroupView } from "./models";

export class QuestionGroupService extends BaseService<
QuestionGroup,
QuestionGroupRequest,
QuestionGroupResponse,
QuestionGroupView
> {
     constructor() {
          super(ENV.API_URL, 'question-groups');
     }
}

export const questionGroupService = new QuestionGroupService();